packages/frontend/src/pages/layer3s/props/getProjectHeader.ts

```diff
@@ -0,0 +1,72 @@
+import { layer2s, Layer3, ProjectLinks } from '@l2beat/config'
+
+import { ProjectLink } from '../../../components/icons'
+import { isAnySectionUnderReview } from '../../../utils/project/isAnySectionUnderReview'
+import { ProjectHeaderProps } from '../view/ProjectHeader'
+
+export function getProjectHeader(project: Layer3): ProjectHeaderProps {
+  return {
+    icon: `/icons/${project.display.slug}.png`,
+    title: project.display.name,
+    titleLength: getTitleLength(project.display.name),
+    purpose: project.display.purpose,
+    technology: project.display.category,
+    links: getLinks(project.display.links),
+    isArchived: project.isArchived,
+    isUpcoming: project.isUpcoming,
+    isUnderReview: project.isUnderReview,
+    showProjectUnderReview: isAnySectionUnderReview(project),
+    warning: project.display.headerWarning,
+    hostChain: layer2s.find((l) => l.id === project.hostChain)?.display.name,
+  }
+}
+
+function getTitleLength(name: string): 'long' | 'very-long' | undefined {
```

Comment:
why not just name.length with some arbitrary boundries ?

Reply:
+1